src/i18n/en-events.ts

```diff
@@ -43,6 +44,7 @@ const eventsMessageDescriptors = {
       ${ColonyAndExtensionsEvents.PayoutClaimed} {{agent} claimed a payout of {amount} {tokenSymbol} from funding pot {fundingPot}}
       ${ColonyAndExtensionsEvents.ColonyMetadata} {{agent} changed Colony metadata to {metadata}}
       ${ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots} {{agent} transferred {amount} {tokenSymbol} between pots}
+      ${ColonyAndExtensionsEvents.ColonyRoleSet} {{agent} {roles} in {domain} to {recipient}}
```

Comment:
This does not work well for "removed", since it says "to", which only applies to adding. In the case of removing it should say "from"

```suggestion
      ${ColonyAndExtensionsEvents.ColonyRoleSet} {{agent} {roles} in {domain} from {recipient}}
```

Here's an example:
![Screenshot from 2021-02-10 17-37-07](https://user-images.githubusercontent.com/1193222/107533146-4ed24880-6bc7-11eb-9853-9fb5cf808746.png)

Consider creating two messages descriptors Id for this instance